stakepooldclient/stakepooldclient.go

```diff
@@ -688,3 +688,16 @@ func (s *StakepooldManager) GetStakeInfo() (*pb.GetStakeInfoResponse, error) {
 	}
 	return nil, errors.New("GetStakeInfo RPC failed on all stakepoold instances")
 }
+
+func (s *StakepooldManager) GetColdWalletExtPub() (*pb.GetColdWalletExtPubResponse, error) {
```

Comment:
The issue this PR is set out to fix says to check the `ColdWalletExtPub` across ALL config files, which means this gRPC request need to be made to ALL stakepoold backend instances and the result from each backend compared with the configuration set in `dcrstakepool.conf`.